patches/rcgen+0.9.3.patch

```diff
@@ -0,0 +1,58 @@
+diff --git a/src/lib.rs b/src/lib.rs
+index 565b3d6..23998c3 100644
+--- a/src/lib.rs
++++ b/src/lib.rs
+@@ -1500,6 +1500,9 @@ impl KeyPair {
+ 		} else if alg == &PKCS_RSA_PSS_SHA256 {
+ 			let rsakp = RsaKeyPair::from_pkcs8(pkcs8)?;
+ 			KeyPairKind::Rsa(rsakp, &signature::RSA_PSS_SHA256)
++		} else if alg == &PKCS_RSA_PSS_SHA384 {
++			let rsakp = RsaKeyPair::from_pkcs8(pkcs8)?;
++			KeyPairKind::Rsa(rsakp, &signature::RSA_PSS_SHA384)
+ 		} else {
+ 			panic!("Unknown SignatureAlgorithm specified!");
+ 		};
+@@ -1886,6 +1889,7 @@ impl SignatureAlgorithm {
+ 			&PKCS_RSA_SHA384,
+ 			&PKCS_RSA_SHA512,
+ 			//&PKCS_RSA_PSS_SHA256,
++			//&PKCS_RSA_PSS_SHA384,
```

Comment:
Why this?

Reply:
Just to keep track of the changes required when merging into rcgen